assets/src/js/country_selector.js

```diff
@@ -0,0 +1,50 @@
+// The script basically add/remove the tabindex attribute within the country selector
+// It helps to improve SEO
+export const setupCountrySelector = () => {
+  const targetNode = document.querySelector('#country-selector');
+  const countriesList = document.querySelector('.countries-list');
+  const footerMenu = document.querySelector('.footer-menu');
+  const footerSocialMedia = document.querySelector('.footer-social-media');
+  const largeAndUpScreens = '(min-width: 992px)';
+
+  const switchOrders = (evt = null) => {
+    const event = evt ? evt : window.matchMedia(largeAndUpScreens);
+    if (event.matches) {
```

Comment:
The `insertBefore` call always executes regardless of the current order, even if the elements are already in the correct position:
```suggestion
    if (event.matches && footerMenu !== footerMenu.parentNode.firstChild) {
```

Reply:
Sorry! I'm not sure if I understand correctly your suggestion 🙏 .

When the event matches, the `footerMenu` and `footerMenu.parentNode.firstChild` are never the same. This part is invoked right only the window size is being resized and following this `min-width: 992px`, and not every time you resize even if the size is `1024px` or `1600px`.